corehq/apps/sms/tasks.py

```diff
@@ -263,7 +263,7 @@ def current_usage(self):
     @property
     def daily_limit(self):
         if self.domain_object:
-            return self.domain_object.daily_outbound_sms_limit
+            return self.domain_object.get_daily_outbound_sms_limit()
```

Comment:
What do you think of keeping the existing interface by renaming `get_daily_outbound_sms_limit` to `daily_outbound_sms_limit` and add `@property`?

Reply:
i initially was going to do that but then i felt that making a new name would be better